src/components/Button.js

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ buttonName, color, wide }) => (
-  <div className={`button ${color} ${wide ? 'large-button' : ''}`}>
-    <p className="button-text">{buttonName}</p>
-  </div>
-);
+const Button = ({
+  clickHandler, buttonName, color, wide,
+}) => {
+  const handleClick = buttonName => clickHandler(buttonName);
+
+  return (
+    <button
+      type="submit"
+      className={`button ${color} ${wide ? 'large-button' : ''} button-text`}
+      onClick={() => handleClick(buttonName)}
```

Comment:
Nice! And since you have `buttonName` as a prop in the scope of the function, you could use it instead of passing it as an argument to handleClick and then you just assign handleClick to the onClick event:

```js
  const handleClick = () => clickHandler(buttonName);

  return (
    <button
      type="submit"
      className={`button ${color} ${wide ? 'large-button' : ''} button-text`}
      onClick={handleClick}
```

Reply:
done!